byway/src/main/webapp/routepage.html

```diff
@@ -7,15 +7,22 @@
     <script src="https://polyfill.io/v3/polyfill.min.js?features=default"></script>
     <script src="base.js"></script>
     <script src="routepage.js"></script>
+    <script src="recommendations.js"></script>
     <script
       src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBg6D1U0U2L7VlbKH7Bkh-ds6wdUaLzroM&callback=initMap&libraries=places,geometry"
       defer></script>
     <script src="utils.js"></script>
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
```

Comment:
Why do you need to pull in bootstrap css from a CDN when you already have it stored locally under `byway/src/main/webapp/bootstrap/all.css`? Same for the fonts link above this one.

Reply:
I removed from this PR and will configure the bootstrap/styling as needed. Also, the font links and bootstrap folder above were set up by Justine, and she plans on cleaning it up in another PR.